python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -464,8 +474,8 @@ async def latiss_acquire(self):
                         z=-self.manual_focus_offset
                     )
                     self.manual_focus_offset_applied = False
-                self.log.exception(e)
-                raise e
+                self.log.exception("Centroid finding algorithm was unsuccessful.")
```

Comment:
I think you want to use `self.log.error` here... exception is for when you are logging an exception that was raised before.